Version.xcconfig

```diff
@@ -12,3 +12,4 @@ CURRENT_PROJECT_VERSION = 57
 
 // Optional override
 #include? "VersionOverride.xcconfig"
+#include? "../VersionOverride.xcconfig"
```

Comment:
I think this might not be needed, as long as the VersionOverride.xcconfig file is added to the workspace. This change may break the way Tidepool Loop lays out its workspace.

Reply:
For DIY Loop, we want to pick up VersionOverride in LoopWorkspace so we need a "../" to find it.

Do I need to reverse the order? Then if Tidepool has a VersionOverride in Loop, it would get picked up and override whatever is in the LoopWorkspace folder.

It would help if I knew how Tidepool lays out their workspace. Is there a link?

Reply:
Tidepool's workspace is private as it references undisclosed partner information.  I just looked at how the resolution was happening for Tidepool Loop, and it's via a symlink that is put into the Loop directory at build configuration time. We can keep your change, and I'll update the Tidepool scripts to change where they put the override file. 

If we're moving this, then we can also remove the file from the .gitignore.  LoopOverride.xcconfig also behaves in a similar way, but we can leave this as is for now.